src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import lotto.util.RandomUtil;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private static final String STRING_NUMBERS_DELIMITER = ",";
+    private static final int NUMBER_COUNT = 6;
+
+    private final List<LottoNumber> lottoNumbers;
+
+    public Lotto() {
+        this(pickRandomNumber());
+    }
+
+    private static List<LottoNumber> pickRandomNumber() {
+        Set<LottoNumber> tmpLottoNumbers = new HashSet<>();
+        while (tmpLottoNumbers.size() < NUMBER_COUNT) {
+            tmpLottoNumbers.add(LottoNumber.valueOf(RandomUtil.randomInt(LottoNumber.MAX_VALUE)));
+        }
+
+        List<LottoNumber> lottoNumbers = new ArrayList<>(tmpLottoNumbers);
+        Collections.sort(lottoNumbers);
+        return lottoNumbers;
+    }
```

Comment:
다음과 같이 정적 팩터리 메서드를 사용해볼 수도 있을 것 같아요 😄 
```suggestion
    public static Lotto pickRandomNumber() {
        Set<LottoNumber> tmpLottoNumbers = new HashSet<>();
        while (tmpLottoNumbers.size() < NUMBER_COUNT) {
            tmpLottoNumbers.add(LottoNumber.valueOf(RandomUtil.randomInt(LottoNumber.MAX_VALUE)));
        }

        List<LottoNumber> lottoNumbers = new ArrayList<>(tmpLottoNumbers);
        Collections.sort(lottoNumbers);

        return new Lotto(lottoNumbers);
    }
```